parameter/FloatingPointParameterType.cpp

```diff
@@ -138,7 +139,8 @@ bool CFloatingPointParameterType::toBlackboard(
             return false;
         }
 
-        float fData = reinterpret_cast<const float &>(uiValue);
+        float fData;
+        utility::binaryCopy(uiValue, fData);
```

Comment:
That's what I originally did. But consider line 177: I would find dangerous using an explicit template instantiation because, if the type of `uiValue` changes, the code might still compile but will not behave as intended.